adb_graphics/figures/maps.py

```diff
@@ -372,6 +378,34 @@ def _draw_contours(self, ax, not_labeled):
                     print(f'Cannot add contour labels to map for {self.field.short_name} \
                             {self.field.level}')
 
+    def _draw_scatter(self, ax):
+
+        ''' Plot dots at locations on the map that meet a threshold. '''
+
+        field = self.field
+        levels = self.field.clevs
+        alpha_val = 0.75 # set opacity
+        vals = self.field.values()
+        colors = ['white', 'lightskyblue', 'darkblue', 'green', 'darkorange', \
+                  'indianred', 'firebrick']
+
+        ci = copy.copy(vals)
+        ci = np.full_like(ci, colors[0], dtype='object')
+        ci = np.where((vals > levels[0]) & (vals <= levels[1]), colors[1], ci)
+        ci = np.where((vals > levels[1]) & (vals <= levels[2]), colors[2], ci)
+        ci = np.where((vals > levels[2]) & (vals <= levels[3]), colors[3], ci)
+        ci = np.where((vals > levels[3]) & (vals <= levels[4]), colors[4], ci)
+        ci = np.where((vals > levels[4]) & (vals <= levels[5]), colors[5], ci)
+        ci = np.where(vals > levels[5], colors[6], ci)
```

Comment:
I moved the colors array definition to specs.py, and the levels are designated in default_specs.py.  I left the rest here, since the field is needed to assign the colors.